### PR TITLE
feat: flows predefined list selections

### LINKED ISSUE #2308: Notebooks: I can select a notebook preset from the notebook list page
We would like help users discover different ways to use notebooks by adding a section at the top of the notebooks list page with links to predefined notebooks.

Predefined Notebooks to link to: 
- **New Notebook** - Metric selector + simple table + graph (what we currently use for Create Notebook)
- **Set an Alert** - Metric selector + simple table + alert panel
- **Schedule a Task** - Metric selector + simple table + schedule panel
- **Write a Flux Script** - Flux panel + simple table
- **Blank Notebook** - no panels

Prototype: https://www.figma.com/proto/a7qLfWdlUDrhab9aWIxE65/Create-an-Alert-from-Notebook?node-id=772%3A8442&scaling=min-zoom&page-id=662%3A5686&starting-point-node-id=772%3A8225&show-proto-sidebar=1

Images to include:

Set an Alert Image

Schedule a Task Image

Write a Flux Script Image

Blank Notebook Image

Design: @juliajames12 

Design file: https://www.figma.com/file/a7qLfWdlUDrhab9aWIxE65/Create-an-Alert-from-Notebook?node-id=668%3A5337

### PR DESCRIPTION
**Pull request by ShmuelLotman**
Closes #2308

Introduces notebooks prefab panels that allow for selection of specific notebook type 

https://user-images.githubusercontent.com/29473622/129126134-b803daf1-0288-4378-9d1f-3aded67e2339.mov

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 62 | :yellow_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2330)

**Comment by juliajames12** [comment]
@ShmuelLotman This is looking great! Could we just add a small border around 1px in the g5-pepper color like this:

Screen Shot 2021-08-12 at 10 47 04 AM

File: src/flows/components/FromTemplatePage.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ const Template: FC = () => {
 
     add(hydrate(TEMPLATES[params[0]].init.apply(this, params.slice(1)))).then(
       id => {
-        history.push(`/orgs/${org.id}/notebooks/${id}`)
+        history.replace(`/orgs/${org.id}/notebooks/${id}`)
```
Comment (drdelambre) [inline src/flows/components/FromTemplatePage.tsx:35]: :+1:

**Review by drdelambre — CHANGES_REQUESTED** [review]
more comments coming, but first off use the clockface padding definitions

**Comment by drdelambre** [comment]
the "what is a notebook" is in the wrong place, which when fixed will change your div nesting which will remove the need for those random hardcoded values you put in the css

File: src/flows/templates/types/flux.ts [hunk 2]
```diff
@@ -6,38 +6,40 @@ export default register =>
...
-          queries: [
-            {
-              text:
-                '// Uncomment the following line to continue building from the previous cell\n// __PREVIOUS_RESULT__\n',
-              editMode: 'advanced',
-              builderConfig: {
-                buckets: [],
-                tags: [],
-                functions: [],
+      spec: {
```
Comment (drdelambre) [inline src/flows/templates/types/flux.ts:9]: :1st_place_medal:

**Comment by drdelambre** [comment]
this should be wrapped in a feature flag so we can push it out now and keep it off, or else we have to wait for design to audit and fix all of the templates we are using before merging

**Review by drdelambre — APPROVED** [review]
Noice

**Pull request merged**